sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -271,6 +359,9 @@ def from_connection_string(
          For detailed partition context information, please refer to
          :class:`PartitionContext<azure.eventhub.aio.PartitionContext>`.
         :type on_event: Callable[~azure.eventhub.aio.PartitionContext, ~azure.eventhub.EventData]
+        :keyword float max_wait_time: If it's None or 0, `on_event` is never called when no events are received.
+         Otherwise, `on_event` is called wiht param event being `None` every max_wait_time when no events are received.
```

Comment:
Typo: with
We may want to rephrase this a bit, something like:
The maximum interval in seconds that the processor will wait before calling the callback. If no events are received within this interval, the on_event callback will be called with `None`. If this value is set to None or 0 (the default), the callback will not be called until an event is received.

Reply:
Updated both receive and receive_batch docstring for max_wait_time